src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -43,9 +51,9 @@
                        n-basic-valid (if (>= dest-ws-n-len 1) true false)
                        ns-basic-valid (if (>= dest-ws-ns-len 1) true false)]
                    (if-not n-basic-valid
```

Comment:
This line can be replaced with `(if (zero? (count (:name workspace-id)))` (and similarly for the other check), and then you can get rid of 4 lines in the `let`.